src/nyc_trees/apps/survey/templates/survey/partials/reserved_blockface_popup.html

```diff
@@ -40,7 +40,7 @@ <h4 class="modal-title" id="reservation-modal-label">Cancel Reservation</h4>
                 <p>Are you sure you want to cancel?</p>
             </div>
             <div class="modal-footer">
-                <button type="button" class="btn btn-default pull-left" data-dismiss="modal">Keep Reservation</button>
+                <button type="button" class="btn btn-plain pull-left" data-dismiss="modal">No</button>
```

Comment:
When I hover over this button, the text turns the same color as the background, making it "invisible"